src/App.js

```diff
@@ -6,24 +6,24 @@ import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
 
+import { PostList } from './components/PostList';
+
+const postsListAll = posts.map((post) => {
+  const postComponent = { ...post };
```

Comment:
Немає жодного відношення до компонента
```suggestion
  const postCopy = { ...post };
```